packages/core/test/testers.test.ts

```diff
@@ -515,6 +515,13 @@ test('test isEnumControl', t => {
       properties: { foo: { type: 'number', enum: [1, 2] } }
     })
   );
+  t.true(
+    isEnumControl(t.context.uischema, {
+      type: 'object',
+      properties: { foo: { const: '1.0' } },
+      required: ['version']
```

Comment:
the property `version` does not exist in this schema, I think this is a left over from the example, I would suggest to remove it